doc/manual/contracts.md

```diff
@@ -444,7 +444,7 @@ but potentially surprising differences.
 
 One concerns open contracts. Merging never requires the presence of specific
 fields: thus, the contract `{bar | String}` attached to `foo` will actually
-behave as an open contract, even if you didn't use `..`. This might or might not
+behave like an open contract, even if you didn't use `..`. This might or might not
```

Comment:
I would tend to think both "behave as" and "behave like" are valid here. Out of curiosity, what is the nuance making "behave like" more adapted here?